Example/Controllers/Extensions/ViewController+MiniAppUserInfoProtocol.swift

```diff
@@ -45,4 +45,8 @@ extension ViewController: MiniAppUserInfoDelegate {
         }
         completionHandler(.success(.init(accessToken: resultToken, expirationDate: resultDate, scopes: resultScopes)))
     }
+
+    func getPoints(completionHandler: @escaping (Result<MAPoints, MASDKError>) -> Void) {
```

Comment:
@laubengaier . 
[MASDKError](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/core/MASDKError.swift#L4) was used initially in SDK which we thought would have been a better option for handling errors but it turned out to have some limitations when we want to pass any error from the host app. And also we wanted to unify the error response which has been sent to mini-app. And we want to add errors & types in our SDK conforms to [MiniAppErrorProtocol](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/core/Models/MiniAppJavascriptErrorInfo.swift#L1)

So please add a new error type in [MiniAppJavascriptErrorInfo.swift](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/core/Models/MiniAppJavascriptErrorInfo.swift) OR add the errors to the existing Error types that conform to `MiniAppErrorProtocol`

CC: @Climbatize @corycaywood

Reply:
@rleojoseph thanks for checking, I've added MASDKPointError with two basic errors failedToConformToProtocol and error because I'm not sure yet what errors to expect. also applied a fix to the issues you've mentioned. please give it another check.